CMakeLists.txt

```diff
@@ -73,10 +73,14 @@ if(BUILD_LIBOSRM)
   add_custom_command(OUTPUT ${BINDING_DIR}/osrm-datastore
                      COMMAND ${CMAKE_COMMAND} -E copy $<TARGET_FILE:osrm-datastore> ${BINDING_DIR}
                      DEPENDS osrm-datastore ${BINDING_DIR})
+  add_custom_command(OUTPUT ${BINDING_DIR}/osrm-components
```

Comment:
If cmake would be called without `BUILD_COMPONENTS=On` this would leave to a dependency that could not be satisfied. This could be wrapped in a conditional, or always set to `On`.